### ISSUE TITLE
feat(material/tabs): Optional MatTabContent reinstantiating when lazy loaded

### ISSUE THREAD

**Issue by jmeinlschmidt** [angular/components#22356]
#### Feature Description

This feature request follows issue #16065 that was closed due to inactivity in 2019.
Currently, when using `MatTabContent`, the content gets **reinstantiated on each tab change**.
I am willing to have an option to configure the lazy loading process to instantiate the content only once, when it gets loaded for the first time.

#### Use Case

Current state makes some performance impact on our application when using heavy forms within `mat-tab` and makes this lazy loading feature unfortunately useless.

**Comment by crisbeto** [comment]
I believe that this is a duplicate of https://github.com/angular/components/issues/19480.
